py/serial_keel.py

```diff
@@ -362,6 +364,14 @@ def endpoint_messages(self, endpoint: Endpoint) -> EndpointMessages:
             )
             raise e
 
+    def get_endpoint(self, endpoints: List[Endpoint], label: str) -> Endpoint:
+        '''
```

Comment:
I think `"""` is used elsewhere in this file.

So I suggest either
1. Change all to `'''`, or
2. Change this to `"""`